allennlp/models/encoder_decoders/nlvr_semantic_parser.py

```diff
@@ -804,13 +993,15 @@ def _compute_new_states(cls,  # type: ignore
         # may be applicable in the future.
         # Note that we're not sorting these probs. We'll score checklists and sort states based on
         # checklist scores later.
-        probs = torch.exp(log_probs)
-        # batch_index -> [(group_index, action_index, action, checklist, score)]
+        considered_action_probs = nn_util.masked_softmax(action_logits, action_mask)
+        all_agenda_action_probs = state.get_agenda_action_probs()
+        # batch_index -> [(group_index, action_index, action, checklist, model_score)]
         next_states_info: Dict[int, List[Tuple[int, int, int, Variable, Variable]]] = defaultdict(list)
-        for group_index, (batch_index, instance_action_probs) in enumerate(zip(state.batch_indices, probs)):
+        for group_index, (batch_index, instance_score, instance_action_probs) in \
+            enumerate(zip(state.batch_indices, state.score, considered_action_probs)):
```

Comment:
This formatting is pretty bad - it makes it look like the `enumerate` is the first line in the `for` block.

Reply:
Done.